### PR TITLE
adds in linter for engineers, design and copy resources

### PR DESCRIPTION
**Pull request by cheapwebmonkey**
This adds links and resources for in-editor linting, plus Design and copywriting resources for accessibility. There is also a card in the backlog to make Trello cards for these items.

### CODE REVIEW

File: _guide/onboarding-staff.md
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md]: @dmundra is it possible to run the remark-linter locally?

Comment (dmundra) [reply _guide/onboarding-staff.md]: I believe it is via docker. I haven't done it so need to produce some steps but the linter script is located in the `lint` folder.

Comment (dmundra) [reply _guide/onboarding-staff.md]: Here are the steps after checking out the repo and branch

```
docker build -t accessibility-linter lint/. # Build the linter docker image
docker run -v $(pwd):/src/. --rm accessibility-linter lint/remark.sh # Run docker mounting the current directory and using the image built in the previous command
```

**Comment by mgifford** [comment]
@cheapwebmonkey My only thought so far is that not everyone uses Figma or a text editor for their work. Would be good to just add (if you use ... then) so that developers aren't trying to figure out Figma & designers aren't trying to figure out what a linter is.

File: .github/workflows/qa.yml
Comment (mgifford) [reply .github/workflows/qa.yml:58]: Sneaky.

**Comment by mgifford** [comment]
@cheapwebmonkey & @dmundra would be good to give this a quick review to see the modifications I proposed make sense & then let's push this into the site. At this point it has been open for too long. We can fix it later if others have concerns. These are great changes. Thanks!

**Review by mgifford — COMMENTED** [review]
Made some changes. Looks good.  Sorry I missed that this was waiting for my review.

**Review by dmundra — APPROVED** [review]
Looks good to me.

File: _guide/onboarding-staff.md
Comment (jaube05) [inline _guide/onboarding-staff.md:77]: Small typo here, need to remove 'a' after 'Take'

File: _guide/onboarding-staff.md
Comment (jaube05) [inline _guide/onboarding-staff.md:61]: Do we have one for phpstorm or a link for other IDE's developers may use

File: _guide/onboarding-staff.md
Comment (mgifford) [reply _guide/onboarding-staff.md:24]: Might depend on the video. Can you find it @jaube05 ? We can always add it in later.  In this case, there's just an extra space that is being nixed, so we don't want to hold up the patch.

Comment (jaube05) [reply _guide/onboarding-staff.md:24]: We dont need to hold up the patch but here is the video https://www.youtube.com/watch?v=Gv1aDEFlXq8&t=4s

Comment (mgifford) [reply _guide/onboarding-staff.md:24]: I was confused by Nunavut @jaube05 — I now totally understand the confusion now. It was https://www.intuit.com employees. 

Yes. That should be in the onboarding. @cheapwebmonkey do you want to add it here or in another patch?

File: _guide/onboarding-staff.md
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md:61]: There is nothing like this that I have been able to find for PHP Storm

File: _guide/onboarding-staff.md
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md:24]: Another patch please

**Review by mgifford — APPROVED** [review]
Looks good to me. @cheapwebmonkey please merge this issue if you're comfortable with it all.

**Pull request merged**
